### ISSUE TITLE
OSGI Console Over telnet doesn't work

### ISSUE THREAD

**Issue by dumindurox777** [wso2/product-apim#8117]
### Description:
We are facing the following error when trying to access the WSO2 OSGi console using Telnet.

```
Exception in thread "Thread-93" java.lang.NoSuchMethodError:
org.apache.felix.service.command.CommandProcessor.createSession(Ljava/io/InputStream;Ljava/io/OutputStream;Ljava/io/OutputStream;)Lorg/apache/felix/service/command/CommandSession;
Exception in thread "Thread-93" java.lang.NoSuchMethodError:
org.apache.felix.service.command.CommandProcessor.createSession(Ljava/io/InputStream;Ljava/io/OutputStream;Ljava/io/OutputStream;)Lorg/apache/felix/service/command/CommandSession; 
at org.eclipse.equinox.console.telnet.TelnetConnection.run(TelnetConnection.java:90)
```

### Steps to reproduce:
1) Start the server as follows.
./wso2server.sh -DosgiConsole=2500
2) Telnet the connection as follows
telnet <ip_address> 2500

### Affected Product Version:

<label>APIM-3.1.0</label>

### Environment details (with versions):
- OS:
- Client:
- Env (Docker/K8s):

---
#### Related Issues:

https://issues.apache.org/jira/browse/FELIX-6021

**Comment by ruwanta** [comment]
OSGI console access via telnet is an undocumented and can not see a real requirement.
Please use SSH on remote server to access the remote console itself. You can do any task via SSH.

I do not see any real requirement on this request.

**Comment by kakash1hatake** [comment]
I disagree, because accessing is officially documented in [equinox](https://www.eclipse.org/virgo/documentation/virgo-documentation-3.6.4.RELEASE/docs/virgo-user-guide/html/ch08.html)

After some debugging I managed to make it working. 
The problem is in the **org.eclipse.equinox.console_1.3.300.v20190516-1504.jar** in the TelnetConnection.java:90
change line 90 from
```
CommandSession session = this.processor.createSession((InputStream)inp, output, output);
```
to
```
CommandSession session = this.processor.createSession((InputStream)inp, (PrintStream)output, (PrintStream)output);
```

**Comment by kakash1hatake** [comment]
The problem occurs also in the APIM-4.0.0

**Comment by kakash1hatake** [comment]
If someone wants to access console via ssh, then additional [components](http://blog.matbooth.co.uk/14-remote-osgi-console-with-equinox.html) needed to be deployed (they are not included into APIM) and additional configuration via properties

* org.apache.sshd.osgi
* org.eclipse.equinox.console.jaas.fragment
* org.eclipse.equinox.console.ssh

**Comment by kakash1hatake** [comment]
@ruwanta It would be nice if this tiny bug was solved.

**Comment by rgl** [comment]
@ruwanta this would make this way easier when running under the container from https://github.com/wso2/docker-apim/tree/4.3.x/dockerfiles/ubuntu/apim. when its running under the container its somewhat hard to reach its terminal.
